step-templates/linux-service-start-stop-restart.json

```diff
@@ -0,0 +1,54 @@
+{
+    "Id": "cc2aa1d1-975b-4ac4-a145-094bbd92a2c9",
+    "Name": "Linux Service - Start, Stop, Restart",
+    "Description": null,
+    "ActionType": "Octopus.Script",
+    "Version": 1,
+    "CommunityActionTemplateId": null,
+    "Packages": [],
```

Comment:
Could you add an author property and remove the communityactiontemplateid property?